janus_core/calculations/md.py

```diff
@@ -532,6 +532,13 @@ def _set_info(self) -> None:
         except ValueError:
             self.dyn.atoms.info["density"] = 0.0
 
+        e_kin = self.dyn.atoms.get_kinetic_energy() / self.n_atoms
+        current_temp = e_kin / (1.5 * units.kB)
+        self.struct.info["temperature"] = current_temp
+
+        if hasattr(self.dyn, "set_temperature") or isinstance(self, NVT_CSVR | NPT_MTK):
```

Comment:
should not be also NVT in there?

Reply:
That should be included by `hasattr(self.dyn, "set_temperature")`.

These are the same checks added in #414 to make sure `_set_target_temperature` is used appropriately.

(I did consider adding a `has_thermostat` property for this condition, but in `_set_target_temperature` `NVT_CSVR` and `NPT_MTK` are handled individually, so it would only be used here)